Remotion/Web/Development.WebTesting/BrowserLog/IgnoreBrowserLogMessageAttribute.cs

```diff
@@ -0,0 +1,51 @@
+// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Remotion.Web.Development.WebTesting.BrowserSession;
+
+namespace Remotion.Web.Development.WebTesting.BrowserLog;
+
+[AttributeUsage(AttributeTargets.Assembly|AttributeTargets.Class|AttributeTargets.Method, AllowMultiple = true)]
+public class IgnoreBrowserLogMessageAttribute : WebTestAttribute
+{
+  public const string PropertyKey = "IgnoreBrowserLogMessage";
+
+  public IgnoreBrowserLogMessageAttribute (string pattern)
+  {
+    Pattern = pattern;
+  }
+
+  public string Pattern { get; }
+
+  public Func<BrowserLogEntry, bool> GetDelegate ()
```

Comment:
given the fact that we're already using a regex pattern, is it helpful to return funcs isntead of a list of regexes? We only use this in one place and it feels like we're over-wrapping this stuff.